youtube_dl/downloader/rtmp.py

```diff
@@ -143,6 +144,8 @@ def run_rtmpdump(args):
             basic_args += ['--conn', conn]
         if protocol is not None:
             basic_args += ['--protocol', protocol]
+        if real_time:
+            basic_args += ['--realtime']
```

Comment:
Works perfectly without `--realtime` for me, downloaded video is fine. Is it really required?